### ISSUE TITLE
4.2.0 throws error when changing order status

### ISSUE THREAD

**Issue by 30knees** [mollie/PrestaShop#277]
### Expected behavior
Changing order status eg from paid to shipped should work.

### Actual behavior
We get an error message:

```
Decoded exception
[2021-01-15T18:55:04+0100] Invalid order ID: 'tr_7gatDqM8Va'. An order ID should start with 'ord_'.
in file modules/mollie/vendor/mollie/mollie-api-php/src/Endpoints/OrderEndpoint.php at line 65
```

### Steps to reproduce the behavior
Select an order and try to change the status.

### Environment
* Thirtybees 1.1.1 - Prestashop 1.6.X
* Mollie module version: 4.2.0
* PHP version: 7.3.25

**Comment by 30knees** [comment]
4.1.3 works - I just reverted.

**Comment by Saimis777** [comment]
Hey @30knees thank you for that,

I'm checking with the team for the solution to this, and will get back to you as soon as possible!

Best regards,
Invertus Support team.

**Comment by Saimis777** [comment]
Hello @30knees 

This issue is fixed on 4.3.0 version which is planned to be released later on. So the only thing we can suggest is a manual fix for the time being, until the new version is released:

“src/Service/IssuerService.php“ line 39

public function getIdealIssuers()
{
   $environment = (int) Configuration::get(Mollie\Config\Config::MOLLIE_ENVIRONMENT);

   $methodId = $this->paymentMethodRepository->getPaymentMethodIdByMethodId(PaymentMethod::IDEAL, $environment);
       $method = new \MolPaymentMethod($methodId);
       $issuersJson = $this->paymentMethodRepository->getPaymentMethodIssuersByPaymentMethodId($methodId);
   $issuers = json_decode($issuersJson, true);
   $issuerList[PaymentMethod::IDEAL] = [];
   $context = Context::getContext();
   foreach ($issuers as $issuer) {
      $issuer['href'] = $context->link->getModuleLink(
         $this->module->name,
         'payment',
         ['method' => $method->id_method, 'issuer' => $issuer['id'], 'rand' => time()],
         true
      );
      $issuerList[PaymentMethod::IDEAL][$issuer['id']] = $issuer;
   }

   return $issuerList;
}

I am closing the issue for now, if anything else will come up, let me know!

Best regards,
Invertus Support team
